homeassistant/components/generic/config_flow.py

```diff
@@ -283,6 +283,14 @@ async def async_step_import(self, import_config) -> FlowResult:
         if self.check_for_existing(import_config):
             return self.async_abort(reason="already_exists")
         errors, still_format = await async_test_still(self.hass, import_config)
+        if errors.get(CONF_STILL_IMAGE_URL) == "template_error":
+            _LOGGER.warning(
```

Comment:
Why do we even care if the template renders to something valid? We can just import it and the errors will show up when it tries to render the initialized entity.

Reply:
For YAML import I don't think it matters to ignore it. For when a user just typed it in the UI, my aim was to tell them if it was invalid with a UI message (so they can see the error on the UI rather than having to watch the log).